ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2980,6 +2981,42 @@ router_isis:
     router_id: < router_id >
 ```
 
+#### Router Traffic Engineering
+
+```yaml
+router_traffic_engineering:
+  router_id:
+    ipv4: < IPv4_address >
+    ipv6: < IPv6_address >
+  segment_routing:
+    colored_tunnel_rib: true
+    policy_endpoints:
+      - address: < IPv4_address | IPv6_address >
+        colors:
+          - value: < Integer 0-4294967295 >
```

Comment:
Either specify it as integer: ```value: < integer >``` or specify the value without quotes: ```value: < 0-4294967295 >```. Same for all the below knobs in this README.